cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -586,7 +577,8 @@ public TestEventsMessagesLinker getTestEventsMessagesLinker()
 	}
 	
 	@Override
-	protected CompletableFuture<Iterable<StoredTestEventMetadata>> doGetRootTestEventsAsync(Instant from, Instant to) throws CradleStorageException
+	protected CompletableFuture<Iterable<StoredTestEventMetadata>> doGetRootTestEventsAsync(Instant from, Instant to,

```

Comment:
Make an overload here and in methods below to keep Cradle API backward-compatible with previous version.

Reply:
These are protected methods. All public methods are already overloaded.